perfplot/main.py

```diff
@@ -247,35 +247,33 @@ def _b(data, kernel, repeat, timer, is_ns_timer, resolution):
     number = 1
     required_timing = 10 * resolution
     min_timing = 0
+
+    # Adapt the number of runs for the next iteration such that the
+    # required_timing is just exceeded. If the required timing and
+    # minimal timing are just equal, `number` remains the same (up
+    # to an allowance of 0.2).
+    allowance = 0.2
```

Comment:
Why put this outside the loop?

Reply:
I thought it's constant and redefined every iteration. :)

Reply:
The gain in "efficiency" does is outweighed by the clarity of the previous setting.

Reply:
oh, I agree.